src/api/contact-form.ts

```diff
@@ -0,0 +1,101 @@
+import { WebClient, LogLevel } from '@slack/web-api';
+
+const TOKEN = process.env.SLACK_BOT_SY_TOKEN;
+
+/**
+ * Easily build this block structure via https://app.slack.com/block-kit-builder
+ */
+const createSlackMessage = ({ name, email, message }) => {
+  return [
+    {
+      type: 'header',
+      text: {
+        type: 'plain_text',
+        text: 'New form submission',
+      },
+    },
+    {
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text: 'We have received a new message.',
+      },
+    },
+    {
+      type: 'section',
+      fields: [
+        {
+          type: 'mrkdwn',
+          text: `*Name*:\n ${name}`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `*E-mail*:\n ${email}`,
+        },
+      ],
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'section',
+      fields: [
+        {
+          type: 'plain_text',
+          text: message,
+        },
+      ],
+    },
+    {
+      type: 'context',
+      elements: [
+        {
+          type: 'mrkdwn',
+          text: 'Sent via _Satellytes Contact Form_',
+        },
+      ],
+    },
+  ];
+};
+
+export default async function handler(req, res) {
+  const { name, email, message, ...honey } = req.body;
+  console.log('receive form data', { name, email, message }, honey);
+
+  if (honey.firstName || honey.phone) {
```

Comment:
Very cool 👍 

Would be interesting to have a log service to see if and how often it works.

Reply:
yeah, do you have an idea for that ?

Reply:
maybe panelbear works on server side? https://panelbear.com/docs/custom-events/